src/bsp/src/stm32/hal/stm32f429zi/USB_DEVICE/App/usbd_cdc_if.h

```diff
@@ -31,7 +31,7 @@ extern "C" {
 #include "usbd_cdc.h"
 
 /* USER CODE BEGIN INCLUDE */
-
+extern USBD_HandleTypeDef hUsbDeviceFS;

```

Comment:
Where is this variable defined? we try to have almost no user code in the generated files if possible

Reply:
added USB_DEVICE define in the stm USB_CDC files